src/index.html

```diff
@@ -4,10 +4,51 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Product cards</title>
+    <title>stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container stars--0">
+      <span class="stars stars__star"></span>
+      <span class="stars stars__star"></span>
+      <span class="stars stars__star"></span>
+      <span class="stars stars__star"></span>
+      <span class="stars stars__star"></span>
+    </div>
+    <div class="container stars--1">
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star"></span>
+      <span class="stars stars__star"></span>
+      <span class="stars stars__star"></span>
+      <span class="stars stars__star"></span>
+    </div>
+    <div class="container stars--2">
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star"></span>
+      <span class="stars stars__star"></span>
+      <span class="stars stars__star"></span>
+    </div>
+    <div class="container stars--3">
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star"></span>
+      <span class="stars stars__star"></span>
+    </div>
+    <div class="container stars--4">
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star"></span>
+    </div>
+    <div class="container stars--5">
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star active"></span>
+      <span class="stars stars__star active"></span>
```

Comment:
in the checklist it says 
`Each star in a block should have a class stars__star and no extra classes or inline styles`
do not use the extra `active` class.  Try using the nth-child pseudo-class.